USFMConverter/Core/ProjectBuilder.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using USFMConverter.Core.ConstantValue;
+using USFMConverter.Core.Data;
+
+namespace USFMConverter.Core
+{
+    public class ProjectBuilder
```

Comment:
This class seems superfluous.  It doesn't seem to add any value over a constructor for the Project class.  I recommend removing it.